js/leaflet_strawberry.js

```diff
@@ -46,8 +46,12 @@
             if (drupalSettings.format_strawberryfield.leaflet[element_id]['initialzoom'] || drupalSettings.format_strawberryfield.leaflet[element_id]['initialzoom'] === 0) {
               $initialzoom = drupalSettings.format_strawberryfield.leaflet[element_id]['initialzoom'];
             }
+            var $scrollWheelZoom = !drupalSettings.format_strawberryfield.leaflet[element_id]['disable_mouse_zoom'];
```

Comment:
@patdunlavey this needs a fall back default. Since it is a new setting and some users might not have updated their configs (and there is no update hook, pretty sure it is not needed also) you should check if the value is defined at all first and test

Reply:
I rewrote this to use something like the methodology used with other settings. It now...
- sets a default for `$scrollWheelZoom` (true)
- checks if `drupalSettings.format_strawberryfield.leaflet[element_id]['disable_mouse_zoom']` is defined
- if so, sets `$scrollWheelZoom` to the "truthy" version of that value (negated)

Reply:
I added
```
    disable_mouse_zoom:
      type: string
      label: "Disable mouse wheel zoom"
```
Let me know if that's not correct (not sure about `type: string`).